test/bdd/fixtures/vc-rest/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - EDV_REST_HOST_URL=${EDV_HOST}:${EDV_PORT}
       - BLOC_DOMAIN=testnet.trustbloc.local:80
       - UNIVERSAL_RESOLVER_HOST_URL=http://uni-resolver-web:8080/1.0/identifiers
+      - LOCAL_STORE_PATH=bdd-test-local-store
```

Comment:
???

Reply:
See my response to your other comment. With that in mind: within the docker container it doesn't really matter where the storage is for these tests, as long as it's writable.

Reply:
no need